yarn.lock

```diff
@@ -26395,11 +26390,6 @@ whatwg-encoding@^1.0.1, whatwg-encoding@^1.0.5:
   dependencies:
     iconv-lite "0.4.24"
 
-whatwg-fetch@3.1.0:
-  version "3.1.0"
-  resolved "https://registry.yarnpkg.com/whatwg-fetch/-/whatwg-fetch-3.1.0.tgz#49d630cdfa308dba7f2819d49d09364f540dbcc6"
-  integrity sha512-pgmbsVWKpH9GxLXZmtdowDIqtb/rvPyjjQv3z9wLcmgWKFHilKnZD3ldgrOlwJoPGOUluQsRPWd52yVkPfmI1A==
-
 whatwg-fetch@>=0.10.0:
```

Comment:
and this one?

Reply:
probably left due to:
```
Found "isomorphic-fetch#whatwg-fetch@3.0.0"
info This module exists because "_project_#@jaegertracing#jaeger-ui-components#recompose#fbjs#isomorphic-fetch" depends on it.
```